warehouse/ingest-core/src/main/java/datawave/ingest/util/GenerateMultipleNumShardsCacheFile.java

```diff
@@ -27,10 +28,12 @@ public class GenerateMultipleNumShardsCacheFile {
     public static void main(String[] args) throws ParseException, AccumuloException, AccumuloSecurityException, TableNotFoundException, IOException {
         AccumuloCliOptions accumuloOptions = new AccumuloCliOptions();
         Options options = accumuloOptions.getOptions();
-        options.addOption(OptionBuilder.isRequired(true).hasArg().withDescription("Config directory path").create(CONFIG_DIRECTORY_LOCATION_OVERRIDE));
-        options.addOption(OptionBuilder.isRequired(false).hasArg().withDescription("Config file suffix").create(CONFIG_SUFFIEX_OVERRIDE));
-        options.addOption(OptionBuilder.isRequired(false).hasArg().withDescription("Multiple numShards cache file path")
-                        .create(MULTIPLE_NUMSHARD_CACHE_FILE_LOCATION_OVERRIDE));
+        // options.addOption(Option.builder("u").argName("Username").hasArg().required().desc("Accumulo username").build());
+        options.addOption(Option.builder(CONFIG_DIRECTORY_LOCATION_OVERRIDE).argName("Config Directory Path").hasArg().required().desc("Config directory path")
+                        .build());
+        options.addOption(Option.builder(CONFIG_SUFFIEX_OVERRIDE).argName("Config Suffix").hasArg().required().desc("Config file suffix").build());
```

Comment:
Seems like the original code had a typo in `CONFIG_SUFFIEX_OVERRIDE`. Maybe change it to `CONFIG_SUFFIX_OVERRIDE` ?